ui/helpers/utils/notification.util.ts

```diff
@@ -309,38 +301,43 @@ export const getNetworkNameByChainId = (
  * - nativeCurrencyName: The name of the native currency.
  * - nativeCurrencySymbol: The symbol of the native currency.
  * - nativeCurrencyLogo: The logo URL of the native currency.
- * - nativeCurrencyAddress: A default address, typically the zero address.
  * - nativeBlockExplorerUrl: The URL of the block explorer associated with the network, if available.
  */
-export function getNetworkDetailsByChainId(chainId?: keyof typeof CHAIN_IDS): {
+export function getNetworkDetailsByChainId(chainId: number): {
```

Comment:
Cleaned up this utility.

The function types were a bit messy, and now the behaviour will attempt our internal CORE logic before falling back to the extension defaults.